src/utils/authenticate/authenticationCalls.js

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+
+export const userLogin = async (state, dispatch, email, password, navigate) => {
+  try {
+    const res = await axios.post("/api/auth/login", {
+      email,
+      password,
+    });
+
+    if (res.status === 200 || res.status === 201) {
+      const { foundUser, encodedToken } = res.data;
```

Comment:
* Here you can try this "200 || 201"
* And you can use optional chaining in `res.status` like this `res?,status`
* Or else it will throw and error